esp/src/eclwatch/LogVisualizationWidget.js

```diff
@@ -0,0 +1,51 @@
+/*##############################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+############################################################################## */
```

Comment:
There is no need for these header blocks anymore

Reply:
@richardkchapman is it ok to remove this, its on all the js files?

Reply:
@GordonSmith I thought were updating copyright date vs. removing block altogether?

Reply:
Its ok to remove all of them as covered by global licence.